src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -9,6 +9,11 @@ define(function (require, exports, module) {
     var Path = require("filesystem/impls/filer/BracketsFiler").Path;
 
     /**
+     * This variable controls whether or not we want scripts to be run in the preview window or not
+     */
```

Comment:
Add one more sentence: `We do this by altering the mime type from text/javascript to text/x-scripts-disabled below.`